evalml/pipelines/time_series_regression_pipeline.py

```diff
@@ -42,6 +43,13 @@ class TimeSeriesRegressionPipeline(TimeSeriesPipelineBase):
     """
 
     problem_type = ProblemTypes.TIME_SERIES_REGRESSION
+
+    NO_PREDS_PI_ESTIMATORS = [
+        ModelFamily.ARIMA,
+        ModelFamily.EXPONENTIAL_SMOOTHING,
+        ModelFamily.PROPHET,
+    ]
```

Comment:
These are the estimators where we currently can't use the recomposed predictions to calculate the prediction intervals.